packages/theme/css/characteristics/actionable-next.css

```diff
@@ -124,4 +124,130 @@
+  --salt-actionable-negative-foreground-disabled: var(--salt-palette-foreground-primary-disabled);
+  --salt-actionable-negative-borderColor: var(--salt-palette-negative);
+  --salt-actionable-negative-borderColor-hover: var(--salt-palette-negative);
+  --salt-actionable-negative-borderColor-active: var(--salt-palette-negative);
+  --salt-actionable-negative-borderColor-disabled: var(--salt-palette-negative-disabled);
+  --salt-actionable-negative-background: var(--salt-palette-alpha-none);
+  --salt-actionable-negative-background-hover: var(--salt-palette-negative-action-hover);
+  --salt-actionable-negative-background-active: var(--salt-palette-negative-action-active);
+  --salt-actionable-negative-background-disabled: var(--salt-palette-alpha-none);
+
+  /* saltButton-negative-transparent */
+  --salt-actionable-negative-subtle-foreground: var(--salt-palette-foreground-primary);
+  --salt-actionable-negative-subtle-foreground-hover: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-negative-subtle-foreground-active: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-negative-subtle-foreground-disabled: var(--salt-palette-foreground-primary-disabled);
+  --salt-actionable-negative-subtle-borderColor: var(--salt-palette-alpha-none);
+  --salt-actionable-negative-subtle-borderColor-hover: var(--salt-palette-negative);
+  --salt-actionable-negative-subtle-borderColor-active: var(--salt-palette-negative);
+  --salt-actionable-negative-subtle-borderColor-disabled: var(--salt-palette-alpha-none);
+  --salt-actionable-negative-subtle-background: var(--salt-palette-alpha-none);
+  --salt-actionable-negative-subtle-background-hover: var(--salt-palette-negative-action-hover);
+  --salt-actionable-negative-subtle-background-active: var(--salt-palette-negative-action-active);
+  --salt-actionable-negative-subtle-background-disabled: var(--salt-palette-alpha-none);
+
+  /* saltButton-positive-solid */
+  --salt-actionable-positive-bold-foreground: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-foreground-hover: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-foreground-active: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-foreground-disabled: var(--salt-palette-foreground-primary-alt);
+  --salt-actionable-positive-bold-borderColor: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-hover: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-active: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-borderColor-disabled: var(--salt-palette-alpha-none);
+  --salt-actionable-positive-bold-background: var(--salt-palette-positive);
+  --salt-actionable-positive-bold-background-hover: var(--salt-palette-positive-action-hover);
+  --salt-actionable-positive-bold-background-active: var(--salt-palette-positive);
```

Comment:
The active state on the 'next' positive button background is incorrect. It should be `--salt-palette-positive-action-active`